src/components/vm/consoles/consoles.tsx

```diff
@@ -283,7 +300,7 @@ export const ConsoleCard = ({
                 onClick={(event) => {
                     let header_height = 53;
                     const title = document.querySelector(`#${vmId(vm.name)}-consoles .pf-v6-c-card__header-main`);
-                    if (title) {
+                    if (title && title instanceof HTMLElement) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)